packages/file-asset-apis/src/s3/createS3Client.ts

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs-extra';
+import { Agent } from 'https';
+import { S3Client as BaseClient } from '@aws-sdk/client-s3';
+import { NodeHttpHandler } from '@aws-sdk/node-http-handler';
+import type { S3ClientConfig as baseConfig } from '@aws-sdk/client-s3';
+import { debugLogger, has } from '@terascope/utils';
+import type { S3Client } from './client-types';
+
+export interface S3ClientConfig extends baseConfig {
+    sslEnabled?: boolean,
+    certLocation?: string,
+    httpOptions?: object,
+    secretAccessKey?: string,
+    accessKeyId?: string
+}
+
+export async function createS3Client(
```

Comment:
No, it was probably wrong to name the function to create an elasticsearch connection to be called `createClient` as the name itself if unclear. Which client is it? You may think it sufficient since you are pulling from `elasticsearch-store`. However, in locations that we have it in spaces, there would be several createClients pulled and we would have to change the name reference so there would not be naming collisions. I think being more explicit in the name is better and safer